src/Reports/SiteSummary.php

```diff
@@ -55,16 +59,29 @@ public function columns()
     public function getReportField()
     {
         Requirements::css('silverstripe-maintenance/css/sitesummary.css');
-        $grid = parent::getReportField();
-        $config = $grid->getConfig();
+
+        /** @var GridField $gridField */
+        $gridField = parent::getReportField();
+
+        $config = $gridField->getConfig();
+
+        /** @var GridFieldExportButton $exportButton */
+        $exportButton = $config->getComponentByType(GridFieldExportButton::class);
+
+        // Switch for an unformatted name and description over the HTML formatted summary field
+        $exportColumns = [
+            'Name' => 'Name',
+            'Description' => 'Description',
+        ] + $this->columns();
+        unset($exportColumns['Summary']);
```

Comment:
I find this more confusing than the inbuilt `summaryFields` feature.

In conjunction with the previous two comments on this iteration of the PR I feel like leveraging pre-provided functionality to it's maximum is of more benefit.

Comments can always be added to make things clearer.

```
public function getReportField()
{
    $gridField = parent::getReportField();
    $config = $gridField->getConfig();
    $config->addComponents(
        // [...] refresh and link buttons
    );
    // The GridFieldExportButton uses our model's summary_fields by default
    // as opposed to this reports `columns` method above {@see Package::$summary_fields}
    return $gridField;
}
```